packages/stylelint-plugin/tests/fixtures/scss.invalid.scss

```diff
@@ -0,0 +1,18 @@
+.n1 {
```

Comment:
Could you add comments to each invalid case to point out what's invalid about it?

Reply:
Done!